docs/blueprints/README.md

```diff
@@ -37,22 +46,24 @@ However, it is expected that there should be feedback from the engineering team,
 
 ## Template
 
-The design document uses `TEMPLATE.md` as a guide. However, it can be modified when necessary.
+The blueprint uses `TEMPLATE.md` as a guide. However, it can be modified when necessary.
 
 The template has the following sections:
 
-* **Abstract** gives an overview of the design document and what it accomplishes. It does not go into details about the problem.
+* **Abstract** gives an overview of the blueprint and what it accomplishes. It does not go into details about the problem.
 
-* **Background** gives an overview of the background why this design document exists. It describes the problems the design document tries to solve. It describes the goals of the design.
+* **Background** gives an overview of the background why this blueprint exists. It describes the problems the blueprint tries to solve. It describes the goals of the design.
 
 * **Design** contains the solution and its parts. The level of detail should correspond to the level of risk. The more problems a wrong solution would cause, the more detailed should be the description. The design should describe the solution to the degree it is usable by the end-user.
 
 * **Rationale** section describes the alternate approaches and trade-offs. It should be clear why the proposed design was chosen among the alternate solutions.
 
-* **Implementation** describes the steps to complete this design document. It should contain a rough outline of tasks. If necessary, there can be additional details about the changes to the codebase and process.
+* **Implementation** describes the steps to complete this blueprint. It should contain a rough outline of tasks. If necessary, there can be additional details about the changes to the codebase and process.
+
+* **Wrap up** describes the steps needed
 
 * **Open Issues** contains open questions that the author did not know how to solve.
 
 ## Format
 
-The design document is intended to be read by outside of Storj, hence avoid acronyms that aren't common in the general developer community. Prefer simple sentences. Active voice is usually easier to read. Overall, be clear.
+The blueprint is intended to be read by developer contributors outside of Storj, but not the wider public. It is okay to assume some familiarity with the Storj project. Try to avoid acronyms that aren't common in the general developer community. Prefer simple sentences. Active voice is usually easier to read. Overall, be clear.
```

Comment:
I wonder should we clarify expected level of quality? e.g. run the document through http://www.hemingwayapp.com (or Grammarly Pro) and simplify sentences when possible.

Reply:
i'm reluctant to specify beyond what the reviewers approve